[2021-10-18 17:19:44,184] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:19:44,190] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:19:44,191] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:19:44,191] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-10-18 17:19:44,191] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:19:44,215] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2021-10-17T00:00:00+00:00
[2021-10-18 17:19:44,216] {standard_task_runner.py:52} INFO - Started process 5210 to run task
[2021-10-18 17:19:44,222] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-10-17T00:00:00+00:00', '--job-id', '52', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp6zj8iba2', '--error-file', '/tmp/tmp6pkv7fch']
[2021-10-18 17:19:44,222] {standard_task_runner.py:77} INFO - Job 52: Subtask task_4
[2021-10-18 17:19:44,318] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-10-17T00:00:00+00:00 [running]> on host airflowvm
[2021-10-18 17:19:44,421] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-10-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-17T00:00:00+00:00
[2021-10-18 17:19:44,422] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-18 17:19:44,422] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-10-18 17:19:44,429] {subprocess.py:75} INFO - Output:
[2021-10-18 17:19:47,431] {subprocess.py:83} INFO - Command exited with return code 0
[2021-10-18 17:19:47,473] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20211017T000000, start_date=20211018T171944, end_date=20211018T171947
[2021-10-18 17:19:47,497] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-10-18 17:19:47,508] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-18 17:51:25,268] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:51:25,280] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:51:25,280] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:51:25,281] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-10-18 17:51:25,281] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:51:25,297] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2021-10-17T00:00:00+00:00
[2021-10-18 17:51:25,299] {standard_task_runner.py:52} INFO - Started process 14467 to run task
[2021-10-18 17:51:25,304] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-10-17T00:00:00+00:00', '--job-id', '5', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp064yfpht', '--error-file', '/tmp/tmpmsw2gwwe']
[2021-10-18 17:51:25,305] {standard_task_runner.py:77} INFO - Job 5: Subtask task_4
[2021-10-18 17:51:25,408] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-10-17T00:00:00+00:00 [running]> on host airflowvm
[2021-10-18 17:51:25,577] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-10-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-17T00:00:00+00:00
[2021-10-18 17:51:25,578] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-18 17:51:25,578] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-10-18 17:51:25,583] {subprocess.py:75} INFO - Output:
[2021-10-18 17:51:28,586] {subprocess.py:83} INFO - Command exited with return code 0
[2021-10-18 17:51:28,638] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20211017T000000, start_date=20211018T175125, end_date=20211018T175128
[2021-10-18 17:51:28,673] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-10-18 17:51:28,710] {local_task_job.py:151} INFO - Task exited with return code 0
